source/Ship.cpp

```diff
@@ -1927,12 +1927,15 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 	else if(requiredCrew && static_cast<int>(Random::Int(requiredCrew)) >= Crew())
 	{
 		pilotError = 30;
-		if(parent.lock() || !isYours)
-			Messages::Add("The " + name + " is moving erratically because there are not enough crew to pilot it."
-				, Messages::Importance::Low);
-		else
-			Messages::Add("Your ship is moving erratically because you do not have enough crew to pilot it."
-				, Messages::Importance::Low);
+		if(isYours)
```

Comment:
```suggestion
		if(isYours || (personality.IsEscort() && Preferences::Has("Extra fleet status messages")))
```

Reply:
This seems like a good idea. We definitely don't want to know about ships that aren't in our fleet, but ships that *are* in our fleet or among our escorts seems relevant and useful to know.